asa_parser/tests/test_parser.py

```diff
@@ -131,6 +131,15 @@ def test_cpu_detailed(self):
         self.assertIn('["Break down of per-core data path versus control point cpu usage:"', result)
 
     def test_ipsec_stats(self):
+
```

Comment:
It does not look like these merge conflicts resolved correctly.  Somehow your code ended up in the middle of test_ipsec_stats() Let's do this - I'll go ahead and rework your code, create a pull-request, and have you review that.  Sound OK?